topic_benchmark/cli.py

```diff
@@ -22,11 +24,23 @@ def run_cli(
     encoder_model: str = "all-MiniLM-L6-v2", out_path: Optional[str] = None
 ):
     vectorizer = default_vectorizer()
+
     print("Loading Encoder.")
-    encoder = SentenceTransformer(encoder_model)
+    # first try to load encoder from registry
+    # if not found, assume encoder is a sentence transformer
+    try:
+        encoder = encoder_registry.get(encoder_model)
+    except:
+        encoder = SentenceTransformer(encoder_model)
+        warnings.warn(
```

Comment:
Does this need to be a warning? Isn't this more an informative message, than something the user should be concerned about?